packages/rocketchat-lib/tests/server.tests.js

```diff
@@ -0,0 +1,206 @@
+/* eslint-env mocha */
+import 'babel-polyfill';
+import assert from 'assert';
+
+import PasswordPoliceClass from '../server/lib/PasswordPolicyClass';
```

Comment:
typo: the class name is actually `PasswordPolicyClass` and not `PasswordPoliceClass`..